deploy/lib/createFunctions.js

```diff
@@ -29,6 +29,66 @@ module.exports = {
       });
   },
 
+  applyDomains(funcId, customDomains) {
+    // we make a diff to know which domains to add or delete
+    const domainsToCreate = [];
+    const domainsIdToDelete = [];
+    const existingDomains = [];
```

Comment:
I think changing existingDomains to a map would be better, it will simplify the research